test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/RunLoadTest.java

```diff
@@ -78,6 +80,11 @@ public RunLoadTest tps(DoubleSupplier targetTps) {
 		return this;
 	}
 
+	public RunLoadTest setMemoLength(IntSupplier memoLength) {
```

Comment:
Just a quick question, given this function, the memo length is still fixed for one test run. It's not like for each transaction, the memo length can be different, right?  The memo length can only be changed by passing different length parameter for each run.